src/design/page-layout.md

```diff
@@ -2,26 +2,73 @@
 
-On the page, content blocks float to fill the available space, according to the section of the [page layout]({% link design/layout-updates.md %}) where they are assigned to appear. You will discover that if you change the layout from a three-column to a two-column layout, the content of the main area expands to fill the available space, and any blocks that are associated with the unused side bar seem to disappear. However, if you restore the three-column layout, the blocks reappear. This fluid approach, or liquid layout, makes it possible to change the page layout without having to rework the content. If you are used to working with individual HTML pages, you will discover that this modular, “building block” approach requires a different way of thinking.
+On the page, content blocks float to fill the available space, according to the section of the [page layout]({% link design/layout-updates.md %}) where they are assigned to appear. You will discover that if you change the layout from a three-column to a two-column layout, the content of the main area expands to fill the available space, and any blocks that are associated with the unused side bar seem to disappear. However, if you restore the three-column layout, the blocks reappear. This fluid approach, or _liquid layout_, makes it possible to change the page layout without having to rework the content. If you are used to working with individual HTML pages, you will discover that this modular, _building block_ approach requires a different way of thinking.
 
 ![]({% link images/images-ee/storefront-2-column-ee.png %}){: .zoom}
-_Two Column Page Layout_{:.ee-only}
+_Standard two-column with left bar page layout_
 
 ## Configure default layouts
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the left panel under **General**, choose **Web**.
+1. In the left panel under _General_, choose **Web**.
 
-1. Expand the **Default Layouts** section, and do the following:
+1. Expand ![]({% link images/images/btn-expand.png %}) the **Default Layouts** section.
 
-   * Choose the **Default Product Layout** that you want to use for product pages.
-   * Choose the **Default Category Layout** that you want to use for category pages.
-   * Choose the **Default Page Layout** that you want to use for CMS pages.
+   ![]({% link images/images-ee/config-general-web-default-layouts.png %}){: .zoom}
+   [*Default Layouts*]({% link configuration/general/web.md %}#default-layouts)
 
-    ![]({% link images/images-ee/config-general-web-default-layouts.png %}){: .zoom}
-    [*Default Layouts*]({% link configuration/general/web.md %})
+1. Choose the **Default Product Layout** that you want to use for product pages.
+
+   This determines the layout that is used by default for product pages.
+
+   |No layout updates |By default, layout updates are not available for product pages.|
```

Comment:
is "By default," needed for each of theses.   the next table doesn't.

Reply:
I took it out there, I'll do that here too.